pkg/machinery/config/types/v1alpha1/v1alpha1_validation.go

```diff
@@ -441,29 +442,22 @@ func checkBond(b *Bond) error {
 	return result.ErrorOrNil()
 }
 
+//nolint: gocyclo
 func checkWireguard(b *DeviceWireguardConfig) error {
```

Comment:
we should also validate that only one `EnableAutomaticNodes` iface is present

Reply:
done